client/odie/message/index.tsx

```diff
@@ -79,6 +82,42 @@ const ChatMessage = ( { message, scrollToBottom }: ChatMessageProps ) => {
 		setIsFullscreen( ! isFullscreen );
 	};
 
+	const handleWheel = useCallback(
+		( event: WheelEvent ) => {
+			if ( ! isFullscreen ) {
+				return;
+			}
+
+			const element = fullscreenRef.current;
+
+			if ( element ) {
+				const { scrollTop, scrollHeight, clientHeight } = element;
+				const atTop = scrollTop <= 0;
+				const tolerance = 2;
+				const atBottom = scrollTop + clientHeight >= scrollHeight - tolerance;
+
+				// Prevent scrolling the parent element when at the bounds
+				if ( ( atTop && event.deltaY < 0 ) || ( atBottom && event.deltaY > 0 ) ) {
+					event.preventDefault();
+					event.stopPropagation();
+				}
+			}
+		},
+		[ isFullscreen ]
+	);
+
+	useEffect( () => {
+		const fullscreenElement = fullscreenRef.current;
+		if ( fullscreenElement ) {
+			fullscreenElement.addEventListener( 'wheel', handleWheel, { passive: false } );
+		}
+		return () => {
+			if ( fullscreenElement ) {
+				fullscreenElement.removeEventListener( 'wheel', handleWheel );
```

Comment:
I needed to add the hook that way, as React does not offer a way of adding the "passive: false" optino